test/domains/vass/arrays-of-domains.chpl

```diff
@@ -0,0 +1,47 @@
+// This is analogous to "skyline arrays", where an array's elements
+// are domains instead of arrays. Cf.
+//   test/arrays/ferguson/from-iterator/skyline.chpl
+
+// Test a skyline array / jagged array created
+// from a for expression.
```

Comment:
This comment doesn't seem to make sense; it's not a skyline array but an array of domains.